WordPress/Classes/Utility/Media/AsyncImageView.swift

```diff
@@ -145,6 +147,14 @@ final class AsyncImageView: UIView {
         self.errorView = errorView
         return errorView
     }
+
+    override func hitTest(_ point: CGPoint, with event: UIEvent?) -> UIView? {
+        if configuration.passTouchesToSuperview && self.bounds.contains(point) {
```

Comment:
This was needed to make it work in `WPRichTextImage` (`UIControl`, needed to pass gestures to the superview)